packages/rspack/src/ResolverFactory.ts

```diff
@@ -9,29 +9,34 @@ type ResolveOptionsWithDependencyType = Resolve & {
 
 export class ResolverFactory {
 	#binding: binding.JsResolverFactory;
+	// context to load pnp_manifest
+	context: string;
 
 	static __to_binding(
 		resolver_factory: ResolverFactory
 	): binding.JsResolverFactory {
 		return resolver_factory.#binding;
 	}
 
-	constructor() {
+	constructor(context: string) {
```

Comment:
this seems break ResolverFactory api, I'm not sure whether this api is exposed to user side

Reply:
it's indeed a breaking js api of webpack, need find a better way

Reply:
it seem this is not a public api